x-pack/qa/multi-cluster-search-security/legacy-with-basic-license/src/test/resources/rest-api-spec/test/querying_cluster/10_basic.yml

```diff
@@ -352,3 +352,40 @@ teardown:
   - match: { acknowledged:   true }
 
 
+---
+"Search against remote data streams":
+  - do:
+      headers: { Authorization: "Basic am9lOnMza3JpdC1wYXNzd29yZA==" }
+      search:
+        index: '*:simple-data-stream*'
```

Comment:
Should you add `ccs_minimize_roundtrips: true` here to make it clear what you are testing (compared to the next test)?

Reply:
I think we prefer the implicit default; but I pushed https://github.com/elastic/elasticsearch/pull/96469/commits/93fcddb3878412c10aac1a19449a1b2095c3028e.